Gothic-Unity-Project/Gothic-Unity
# Feature/117 refactor scene handling

## Description
Scene handling moved from GUZSceneManager towards separate *SceneManager classes which call LoadScene and UnloadScene directly. This lowered complexity.

## To test
* Set some settings inside GameConfiguration (with/without main menu, spawn at specific VOB)
* Check if the game boots normally and if you're spawned at the right spot with VR Player

## Technical details
Diagram about changes can be found here: https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/wiki/Developer-Hints#scene-loading
